projects/noloan2/app/src/main/java/hasadna/noloan2/NonSwipeableViewPager.java

```diff
@@ -0,0 +1,57 @@
+package hasadna.noloan2;
+
+import android.content.Context;
+import android.support.v4.view.ViewPager;
+import android.util.AttributeSet;
+import android.view.MotionEvent;
+import android.view.animation.DecelerateInterpolator;
+import android.widget.Scroller;
+import java.lang.reflect.Field;
+
+public class NonSwipeableViewPager extends ViewPager {
```

Comment:
Maybe change the name to "LawsuitViewPager" and add a comment about it being non swipe.

Reply:
Thanks. Changed it to `LawsuitViewPagerAdapterNonSwipe`
Comment's in the name itself.

Reply:
I think this name is too long. The fact that it is "non swipe" is not important.
This class is the ViewPageAdapter for the lawsuit.

Reply:
Actually the whole purpose of this extension is for the non swipe.
Otherwise there's no need for this class at all,  just use it as is from the import in the Activity itself.

Oh and my mistake, not adapter, ViewPager.
Anyway.. Shortened the code a little and changed it to `LawsuitViewPager` :)